MONITORING.md

```diff
@@ -0,0 +1,28 @@
+The `kudo-spark-operator` is able to seamlessly integrated with Prometheus, which is installed within [Prometheus operator](https://github.com/coreos/prometheus-operator).
+Integration with other Prometheus distributions, like kube-prometheus, wasn't tested. 
+
+The `prometheus-operator` uses crafted services discovery approach, introducing `ServiceMonitor` kind. 
```

Comment:
Prometheus Operator relies on `ServiceMonitor` kind which describes the set of targets to be monitored. KUDO Spark Operator configures `ServiceMonitor`s for both the Operator and submitted Spark Applications automatically when monitoring is enabled.

Reply:
Thanks!

Reply:
didn't work as a suggestion for some reason, do you think we still need the sentence above?

Reply:
it removed